docs/releases/v2.6.0/sdlc-compliance.md

```diff
@@ -0,0 +1,45 @@
+SSDLC Compliance Report: Atlas Kubernetes Operator Manager v2.6.0
+=================================================================
+
+- Release Creators: s.urbaniak@mongodb.com,igor.karpukhin@mongodb.com,helder.santana@mongodb.com,jose.vazquez@mongodb.com
```

Comment:
Thanks for the credits, but in SDLC parlance the creator is the release driver, usually one or o couple or engineers, not necessarily the whole team.

Reply:
I don't know, the release has this parameter required. Not sure since then though

Reply:
Yes, it is required. What I mean is we do not need to be all in the list, as only the release driver should be there.
For example, I had a joint release I think with Roo at some point, then the rest have been me. As I was the sole release driver on those, it was only my name here. I think authors here should be just you on this one, unless someone else is shadowing or is expected to take over at some point.